src/main/java/tc/oc/pgm/listeners/ServerPingDataListener.java

```diff
@@ -0,0 +1,126 @@
+package tc.oc.pgm.listeners;
+
+import static com.google.common.base.Preconditions.*;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import java.util.UUID;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.server.ServerListPingEvent;
+import tc.oc.pgm.Config;
+import tc.oc.pgm.api.PGM;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.api.match.MatchManager;
+import tc.oc.pgm.api.match.event.MatchLoadEvent;
+import tc.oc.pgm.map.Contributor;
+import tc.oc.pgm.map.MapInfo;
+import tc.oc.pgm.map.PGMMap;
+import tc.oc.pgm.rotation.PGMMapOrder;
+
+public class ServerPingDataListener implements Listener {
```

Comment:
Isn't there already some `PingListener`? Can we have the Motd and this data all in one listener?

Reply:
There is `MotdListener`. https://github.com/TheMolkaPL/PGM-1/blob/de337077ef99a96c0fc4051bc6359061ad049d73/src/main/java/tc/oc/pgm/listeners/MotdListener.java

I felt that it is cleaner to keep them separated. https://github.com/TheMolkaPL/PGM-1/blob/de337077ef99a96c0fc4051bc6359061ad049d73/src/main/java/tc/oc/pgm/listeners/ServerPingDataListener.java